tutorials/polkadot-sdk/testing/spawn-basic-chain.md

```diff
@@ -20,6 +20,35 @@ To successfully complete this tutorial, you must ensure you've first:
 - [Installed Zombienet](/develop/toolkit/parachains/spawn-chains/zombienet/get-started/#install-zombienet){target=\_blank}. This tutorial requires Zombienet version `{{ dependencies.repositories.zombienet.version }}`. Verify that you're using the specified version to ensure compatibility with the instructions.
 - Reviewed the information in [Configure Zombienet](/develop/toolkit/parachains/spawn-chains/zombienet/get-started/#configure-zombienet){target=\_blank} and understand how to customize a spawned network
 
+## Set Up Local Provider
+
+In this tutorial, you will use the the Zombienet , also called native, that enables you to run nodes as local processes in your environment. 
```

Comment:
somethings missing here

Reply:
Looks like this should probably be deleted, since it's covered in the following section